common/tools/mocha-multi-reporter.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+"use strict";
+
+const Mocha = require("mocha");
+const MochaJUnitReporter = require("mocha-junit-reporter");
+
+/**
+ * Usage :
+ * - Meant to be used to leverage Mocha's Spec reporter as well as mocha-junit-reporter for the test runs
+ * - Provide the relative path to this file from your `sdk/service/package-folder/` as the reporter options to mocha
+ * - Example - `--reporter ../../../common/tools/mocha-multi-reporter.js`
+ *
+ * Refer the following docs for more customizations on reporters
+ * https://mochajs.org/api/tutorial-custom-reporter.html
+ *
+ * @class MultiReporter
+ */
+class MultiReporter {
+  constructor(runner) {
+    console.log("Azure SDKs are cool, Custom reporter starts now...");
```

Comment:
Maybe a slightly more serious string? 🤣

Reply:
Sure sure!!  😂
Added while testing the reporter, removing now.